source/templates/application_template.html.md.erb

```diff
@@ -53,6 +53,16 @@ Do you support this application? Help out by [adding your team name][app-yaml] t
   <% end %>
 </ul>
 
+<h3>SSH access</h3>
+
+This application lives on `<%= application.aws_puppet_class %>` machines.
+
+To SSH to one of these machines [via the jumpbox](https://docs.publishing.service.gov.uk/manual/howto-ssh-to-machines-in-aws.html), use this command:
+
+```
+ssh -A -t jumpbox.blue.integration.govuk.digital 'ssh `govuk_node_list -c <%= application.aws_puppet_class %> --single-node`'
```

Comment:
Make this just `jumpbox.integration.govuk.digital` or `jumpbox.integration.publishing.service.gov.uk`